example/ios/example/Info.plist

```diff
@@ -38,20 +38,15 @@
 		<string>UIInterfaceOrientationLandscapeLeft</string>
 		<string>UIInterfaceOrientationLandscapeRight</string>
 	</array>
-	<key>LSApplicationQueriesSchemes</key>
-	<array>
-		<string>facebook</string>
-		<string>whatsapp</string>
-		<string>mailto</string>
```

Comment:
the library won't be able to use these URL schemes on iOS without these, probably should add more actually, instagrams, others?